packages/forms/resources/views/components/textarea.blade.php

```diff
@@ -33,43 +33,47 @@
                 ->class(['fi-fo-textarea overflow-hidden'])
         "
     >
-        <textarea
-            x-ignore
-            @if (FilamentView::hasSpaMode())
-                ax-load="visible"
-            @else
-                ax-load
-            @endif
-            ax-load-src="{{ \Filament\Support\Facades\FilamentAsset::getAlpineComponentSrc('textarea', 'filament/forms') }}"
-            x-data="textareaFormComponent({ initialHeight: @js($initialHeight) })"
-            @if ($shouldAutosize)
-                x-intersect.once="resize()"
-                x-on:input="resize()"
-                x-on:resize.window="resize()"
-            @endif
-            x-bind:style="{ height }"
-            {{ $getExtraAlpineAttributeBag() }}
-            {{
-                $getExtraInputAttributeBag()
-                    ->merge([
-                        'autocomplete' => $getAutocomplete(),
-                        'autofocus' => $isAutofocused(),
-                        'cols' => $getCols(),
-                        'disabled' => $isDisabled,
-                        'id' => $getId(),
-                        'maxlength' => (! $isConcealed) ? $getMaxLength() : null,
-                        'minlength' => (! $isConcealed) ? $getMinLength() : null,
-                        'placeholder' => $getPlaceholder(),
-                        'readonly' => $isReadOnly(),
-                        'required' => $isRequired() && (! $isConcealed),
-                        'rows' => $rows,
-                        $applyStateBindingModifiers('wire:model') => $statePath,
```

Comment:
Does it still work if we keep this line as well, and remove the `x-model`? Otherwise, I think `live(onBlur: true)` and debounce control would break for this component, as these aren't supported by entangle?

You can still pass the entangled prop into the JS as it should keep itself in sync with the `wire:model`, but it can probably be simplified to `state: $wire.$entangle('{{ $statePath }}')` as the state binding modifiers have no effect if you never set the state

Reply:
Appears to be ok. I'm not sure of any side effects of entangle and wire:model, though. Will default to your judgement on this.